index.html

```diff
@@ -249,11 +249,49 @@ <h4>Computation steps</h4>
-              <div><details>
-                <summary>Comment:</summary>
-                <p>By default, <a class="termref">assistive technologies</a> do not relay hidden information, but an author can explicitly override that and include hidden text as part of the <a class="termref">accessible name</a> or <a class="termref">accessible description</a> by using <code>aria-labelledby</code> or <code>aria-describedby</code>. </p>
-              </details></div>
+            <li id="step2A">
+            <ol>
+              <li id="step2A.i">If the <code>current node</code> is <strong>not</strong> <a href="https://html.spec.whatwg.org/multipage/rendering.html#being-rendered">being rendered</a> <strong>and is</strong>:
+                <ul>
+                  <li><strong>Not</strong> directly referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>.</li>
+                  <li><strong>Nor</strong> part of an <code>aria-labelledby</code> or <code>aria-describedby</code> traversal started in <a href="#step2B.ii.b">step 2B.ii.b<a>, where the target of that relation was hidden.</li>
+                  <li><strong>Nor</strong> directly referenced by a native host language text alternative <a class="termref">element</a> (e.g. <code>label</code> in HTML) or <a class="termref">attribute</a>.</li>
+                </ul>
+                Return the empty string.
+                  <div><details>
+                    <summary>Comment:</summary>
+                    <p>By default, <a class="termref">assistive technologies</a> do not relay hidden information, but an author can explicitly override that and include hidden text as part of the <a class="termref">accessible name</a> or <a class="termref">accessible description</a> by using <code>aria-labelledby</code> or <code>aria-describedby</code>. </p>
+                  </details></div>
+                  <div><details>
+                    <summary>Example:</summary>
+                    <p>The following examples show the meaning of the clause "Not part of an <code>aria-labelledby</code> or <code>aria-describedby</code> traversal started in <a href="#step2B.ii.b">step 2B.ii.b<a>, where the target of that relation was hidden".</p>
+                    <p>First, <code>element1</code>'s <a class="termref">accessible name</a> is "hello" because, although <code>element3</code> is hidden, it is part of an <code>aria-labelledby</code> traversal started in <code>element2</code>, which is hidden too.</p>
+                    <pre class="example highlight"><code>&lt;element1 id="el1" aria-labelledby="el2" /&gt;
+&lt;element2 id="el2" hidden&gt;
```

Comment:
I don't think everyone knows that the hidden attribute is exactly the same as display:none. Maybe the example should use style="display:none" ?

Reply:
I considered the most "semantic" version of the attribute would be appropriate, also to state that we refer to multiple kinds of hidden. No strong opinion, though.

Reply:
I settled to `class=hidden`, which I think it's semantic and doesn't tie the example to a specific type of hidden.